health-sdk/sdk/src/main/java/net/gini/android/health/sdk/integratedFlow/PaymentFragment.kt

```diff
@@ -329,6 +329,14 @@ class PaymentFragment private constructor(
                     if (childFragmentManager.fragments.any { it is PaymentComponentBottomSheet }) {
                         return
                     }
+
+                    if (!viewModel.giniInternalPaymentModule.getReturningUser() && viewModel.documentId != null) {
```

Comment:
[Clarification] Does this case only apply to flows with document? For flows without document, we have the same  steps as before?